stellargraph/data/edge_splitter.py

```diff
@@ -480,11 +480,11 @@ def _reduce_graph_by_edge_type_and_attribute(
         attribute and a threshold applied to the latter.
 
         Args:
-            minedges: <list> Spanning tree edges that cannot be removed.
-            p: <float> Factor by which to reduce the size of the graph.
-            edge_label: <str> The edge type to consider.
-            edge_attribute_label: <str> The edge attribute to consider.
-            edge_attribute_threshold: <str> The threshold value; only edges with attribute value larger than the
+            minedges (list): Spanning tree edges that cannot be removed.
+            p (float): Factor by which to reduce the size of the graph.
+            edge_label (str): The edge type to consider.
+            edge_attribute_label (str): The edge attribute to consider.
+            edge_attribute_threshold (str): The threshold value; only edges with attribute value larger than the
```

Comment:
Although these are "optional", they're also not, since an error is thrown if they're passed in with the default value of `None` so I chose to not document as optional.